modules/setting/setting.go

```diff
@@ -390,6 +390,7 @@ var (
 		JWTSecretBase64            string `ini:"JWT_SECRET"`
 		JWTSigningPrivateKeyFile   string `ini:"JWT_SIGNING_PRIVATE_KEY_FILE"`
 		MaxTokenLength             int
+		EnableRedirectURIWildcard  bool `ini:"ENABLE_REDIRECT_URI_WILDCARD"`
```

Comment:
I think we do not need this option. In fact, the wildcards are not valid URL character, they shouldn't exist in existing URLs and won't conflict.

Reply:
So, it should be safe to just make the wildcard matching behavior the default?

Reply:
Just my opinion, either is fine.

Reply:
I agree no need for such option